src/core/authentification/auth.hook.ts

```diff
@@ -0,0 +1,7 @@
+import { useContext } from 'react';
+import { AuthContext } from './AuthProvider';
+
+export const useAuth = () => {
+  const currentUserContext = useContext(AuthContext);
+  return currentUserContext;
```

Comment:
You need to do other actions ? If not maybe an inline return `return useContext(AuthContext);`